index.js

```diff
@@ -1,22 +1,27 @@
 #!/usr/bin/env node
 const { exec } = require('child_process');
 const path = require('path');
+const debug = process.argv[2] !== '';
```

Comment:
This is not setting `debug==false` as default, given `"" !== undefined`. Why not:
```
const verbose = process.argv[2] === 'verbose';
```
Also, it would be nice to re-adjust the params order if the option the `verbose` is passed.
```
const initialPosition = verbose ? 3 : 2;
const scriptVariables = {
  customThemeFilePath: path.join(process.cwd(), process.argv[initialPosition] || './custom-theme.less'),
  generatedThemeFilePath: path.join(process.cwd(), process.argv[initialPosition + 1] || './custom-theme.css'),
  antdLibraryPath: path.join(process.cwd(), process.argv[initialPosition + 2] || './node_modules/antd')
};
```